### PR TITLE
Set correct time with ntpsec

### PR DESCRIPTION
**Pull request by ruslashev**
This adds a package `ntpsec` which automatically synchronizes time from the Internet. While we are here, also prettify the buildroot config to be more readable and manageable.

### CODE REVIEW

File: tp2bmc/configs/tp2bmc_defconfig [hunk 1]
```diff
@@ -60,50 +60,61 @@ BR2_TARGET_UBOOT_BOARD_DEFCONFIG="sun8iw20p1_auto_nand"
...
 BR2_PACKAGE_HOST_GENEXT2FS=y
 BR2_PACKAGE_HOST_GENIMAGE=y
-BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_5_4=y
+BR2_PACKAGE_HOST_MKPASSWD=y
+BR2_PACKAGE_HOST_MTD=y
 BR2_PACKAGE_HOST_MTOOLS=y
 
-BR2_PACKAGE_NSS_MDNS=y
+# Packages
+BR2_PACKAGE_ALSA_UTILS_AMIXER=y
```
Comment (svenrademakers) [inline tp2bmc/configs/tp2bmc_defconfig:75]: we can turn off these packages as well. we dont have any audio capabilities on the board

File: tp2bmc/package/tpi_rs/Config.in [hunk 2]
```diff
@@ -1,7 +1,6 @@
 config BR2_PACKAGE_TPI_RS
 	bool "tpi_rs"
 	depends on BR2_PACKAGE_HOST_RUSTC_TARGET_ARCH_SUPPORTS
-	depends on BR2_PACKAGE_SQLITE
```
Comment (svenrademakers) [inline tp2bmc/package/tpi_rs/Config.in:4]: `tpi_rs` has a runtime dependency on SQLITE, which i think we should express. But perhaps we want to use `select` here? 

> Use a select type of dependency for dependencies on libraries. These dependencies are generally not obvious and it therefore make sense to have the kconfig system ensure that the dependencies are selected. For example, the libgtk2 package uses select BR2_PACKAGE_LIBGLIB2 to make sure this library is also enabled. The select keyword expresses the dependency with a backward semantic.

**Comment by svenrademakers** [comment]
fixes #51

File: tp2bmc/package/tpi_rs/Config.in (hunk 2 above)
Comment (ruslashev) [reply tp2bmc/package/tpi_rs/Config.in:4]: It is mentioned in the commit message:

> SQLite is built by a crate libsqlite3-sys, on which we depend from sqlx

File: tp2bmc/configs/tp2bmc_defconfig (hunk 1 above)
Comment (ruslashev) [reply tp2bmc/configs/tp2bmc_defconfig:75]: Anything else other than all of `*_ALSA_UTILS*`?

Comment (svenrademakers) [reply tp2bmc/configs/tp2bmc_defconfig:75]: as far as i can quickly scan, this is it

Comment (ruslashev) [reply tp2bmc/configs/tp2bmc_defconfig]: Done

**Review by svenrademakers — APPROVED** [review]

**Pull request merged**